server/autotest_server/testers/ai/settings_schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "type": "object",
+  "required":
+    ["tester_type", "test_data"],
+  "properties": {
+    "tester_type": {
+      "type": "string",
+      "enum": ["ai"]
+    },
+    "test_data": {
+      "title": "Test Groups",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "type": "object",
+        "required": ["config", "timeout", "output", "title", "category"],
```

Comment:
"category" is not required by the other testers, and so should not be required here (it's not a bad idea to require it, but we should be consistent with the other testers)